packages/normalizers/src/withNormalizeTypes.ts

```diff
@@ -28,24 +28,17 @@ export const withNormalizeTypes = <
 
           if (node) {
             if (strictType && isElement(node) && node.type !== strictType) {
-              setElements(
-                editor,
-                { type: strictType },
-                {
-                  at: path,
-                }
-              );
+              setElements(editor, editor.blockFactory({ type: strictType }), {
```

Comment:
Using `blockFactory` in `setElements` (an alias for `setNodes`) might cause problems, since `blockFactory` returns a full element including `children`, but `setNodes` cannot be used to set the children of an element.